roxie/ccd/ccdstate.cpp

```diff
@@ -1569,19 +1569,27 @@ class CRoxieDaliQueryPackageManager : public CRoxieQueryPackageManager, implemen
 
     virtual ISafeSDSSubscription *linkIfAlive() override { return isAliveAndLink() ? this : nullptr; }
 
-    virtual void notify(SubscriptionId id, const char *xpath, SDSNotifyFlags flags, unsigned valueLen, const void *valueData)
+    virtual void notify(SubscriptionId id, const char *xpath, SDSNotifyFlags flags, unsigned valueLen, const void *valueData) override
     {
-        reload(false);
-        daliHelper->commitCache();
+        //Mark this queryset as potentially modified - and then request an incremental reload
+        dirty = true;
+        globalPackageSetManager->requestReload(false, false, true);
```

Comment:
I can't remember how this works, but are you sure that this is the correct packageSetManager to be requesting a reload for?

Reply:
Maybe there is only one. But I thought notifications were done at a more specific level